Automattic/jetpack
# Recommendations: update links to Jetpack Mobile

Fixes #26657

## Proposed changes:

Let's link to jetpack.com/app instead of jetpack.com/mobile. It provides a better UX, since /app then redirects to the appropriate app store for your device.

**To do**

- [x] I need to resize the QR codes, they're too big right now, I picked the wrong size.
- [x] We also need to update the `jetpack-plugin-recommendations-mobile-app-component` redirect link in mc.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Dashboard > Recommendations
* Skip to the last screen.
* With your phone, scan the QR code; it should redirect you to the app store for your device.

You can try scanning the different QR codes I refreshed directly from this PR to ensure that they all work.
